src/components/MsvcSuccess/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { h, FunctionComponent } from 'preact'
+import classNames from 'classnames'
+import { Button } from '@onfido/castor-react'
+
+import { useLocales } from '~locales'
+import theme from 'components/Theme/style.scss'
+import { trackComponent } from '../../Tracker'
+import PageTitle from '../PageTitle'
+import ScreenLayout from '../Theme/ScreenLayout'
+import style from './style.scss'
+import { qrCode } from './assets'
+import { performHttpReq, HttpRequestParams } from '~utils/http'
+
+const pin = 1567
+
+type MsvcSuccessActionsProps = {
+  onFinishFlow: () => void
+}
+
+const finishFlow = (rpToken: string) => {
+  const completeUrl =
+    process.env.NODE_ENV === 'development'
+      ? 'http://localhost:3000'
+      : 'https://microsoft-authenticator-backend.us.onfido.com'
+  const options: HttpRequestParams = {
+    endpoint: `${completeUrl}/complete`,
+    // this should be passed in as a set up variable
+    token: `Bearer ${rpToken}`,
+  }
+  performHttpReq(
+    options,
+    () => {
+      console.log('succcessfully performed request')
+      window.location.replace(completeUrl)
+    },
+    () => {
+      console.log('something went wrong')
+    }
+  )
+}
+
+const MsvcSuccessActions: FunctionComponent<MsvcSuccessActionsProps> = ({
+  onFinishFlow = finishFlow,
+}) => {
+  const { translate } = useLocales()
+
+  return (
+    <div className={theme.contentMargin}>
+      <Button
+        variant="primary"
+        className={classNames(theme['button-centered'], theme['button-lg'])}
+        onClick={() => {
+          onFinishFlow
+        }}
+        data-onfido-qa="MsvcSuccess-next-btn"
```

Comment:
A quick search through the codebase shows that `data-onfido-qa` uses `camelCase` mixed with `kebob-case`, rather than `PascalCase`. So, this should probably be `msvcSuccess-next-btn`.